pkg/cmd/codespace/create.go

```diff
@@ -89,6 +91,37 @@ func (a *App) Create(ctx context.Context, opts createOptions) error {
 		branch = repository.DefaultBranch
 	}
 
+	devContainerPath := opts.devContainerPath
+
+	// now that we have repo+branch, we can list available devcontainer.json files (if any)
+	if len(opts.devContainerPath) < 1 {
+		a.StartProgressIndicatorWithLabel("Fetching devcontainer.json files")
+		devContainerPaths, err := a.apiClient.ListDevContainers(ctx, repository.ID, branch, 100)
+		if err != nil {
+			return fmt.Errorf("error getting devcontainer.json paths: %w", err)
+		}
+		a.StopProgressIndicator()
+
+		if len(devContainerPaths) > 0 {
+			devContainerPathQuestion := &survey.Question{
+				Name: "devContainerPath",
+				Prompt: &survey.Select{
+					Message: "Devcontainer.json file:",
```

Comment:
Minor UI note: seeing `devcontainer.json` in a sentence looks weird because of the period character. Could we possibly standardize on calling them "devcontainer file"?

Reply:
After talking with product it sounds like they want to standardize on `Devcontainer definition` so I'll change that.